### ISSUE TITLE
why multiplied by 64 after cv2.resize() in image.py??

### ISSUE THREAD

**Issue by guoqiang01486** [leeyeehoo/CSRNet-pytorch#29]
hi,leeyeehoo,thank u for the released codes.
In image.py, I don't understand  the meaning  of the code" target = cv2.resize(target,(target.shape[1]/8,target.shape[0]/8),interpolation = cv2.INTER_CUBIC)*64 ",maybe the reason is the resize scale is 1/8,but *64 means the pixels*64,it is irrelevant to the scale . or it only increases the values of pixels,if so ,why *64  not others. I just want to know why?

**Comment by ZhengPeng7** [comment]
> hi,leeyeehoo,thank u for the released codes.
> In image.py, I don't understand the meaning of the code" target = cv2.resize(target,(target.shape[1]/8,target.shape[0]/8),interpolation = cv2.INTER_CUBIC)*64 ",maybe the reason is the resize scale is 1/8,but _64 means the pixels_64,it is irrelevant to the scale . or it only increases the values of pixels,if so ,why *64 not others. I just want to know why?

Because 8 * 8 = 64.
Actually, `resize + *64` should be a simplified method of the convolution that sums the 8x8 grid, which I did in my own implementation.

**Comment by hellowangqian** [comment]
@ZhengPeng7 The operation "resize and then *64" will change the count value (target.sum()) though not significantly.

**Comment by ZhengPeng7** [comment]
Certainly, that's why I called it the 'simplified one', and using convolution to do the count is the precise answer.

**Comment by elharroussomar** [comment]
its give errors why for me. can any one help.

  target = cv2.resize(target,(target.shape[1]/8,target.shape[0]/8),interpolation = cv2.INTER_CUBIC)*64
TypeError: integer argument expected, got float

**Comment by Akbarkhan87** [comment]
> its give errors why for me. can any one help.
> 
> target = cv2.resize(target,(target.shape[1]/8,target.shape[0]/8),interpolation = cv2.INTER_CUBIC)*64
> TypeError: integer argument expected, got float

Just type 
target = cv2.resize(target,(target.shape[1]//8,target.shape[0]//8),interpolation = cv2.INTER_CUBIC)*64
